simulation/components/GarrisonHolder.js

```diff
@@ -77,17 +79,18 @@ GarrisonHolder.prototype.Init = function()
 	if (this.template.VisibleGarrisonPoints)
 	{
 		let points = this.template.VisibleGarrisonPoints;
-		for (let point in points)
+		for (let point in points) {
 			this.visibleGarrisonPoints.push({
 				"offset": {
 					"x": +points[point].X,
 					"y": +points[point].Y,
 					"z": +points[point].Z,
-					"allowedClasses": points[point].VisibleGarrisonAllowedClasses
+					"allowedClasses": points[point].VisibleGarrisonAllowedClasses ? points[point].VisibleGarrisonAllowedClasses : "Infantry+Ranged"
 				},
```

Comment:
If you go that road templates should be updated as well

Reply:
I am not sure what do you mean.